backend/index.js

```diff
@@ -19,4 +20,36 @@ app.get("/", (req, res) => {
   res.send("Hello world!");
 });
 
+// #1 - get all events
+app.get("/api/events", async (req, res) => {
+  try {
+    const events = await Event.find({});
+    res.send(events);
+  } catch (error) {
+    res.status(500).send(error.message);
+    console.log(`error is ${error.message}`);
+  }
+});
+
+// #2 - add an event
+app.post("/api/recipe", async (req, res) => {
```

Comment:
Bootcamp code spotted :)
```suggestion
app.post("/event/add", async (req, res) => {
```